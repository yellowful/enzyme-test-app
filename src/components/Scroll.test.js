import React from "react";
import { shallow } from "enzyme";
import Scroll from "./Scroll";

it('test of Scroll',()=>{
    expect(shallow(<Scroll />)).toMatchSnapshot();
})