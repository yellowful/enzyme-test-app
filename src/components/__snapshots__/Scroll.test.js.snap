// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of Scroll 1`] = `
<div
  style={
    Object {
      "border": "5px solid black",
      "height": "800px",
      "overflow": "scroll",
    }
  }
/>
`;
